roles/coral_credits/defaults/main.yaml

```diff
@@ -0,0 +1,20 @@
+---
+
+coral_credits_chart_repo: https://stackhpc.github.io/coral-credits
+coral_credits_chart_name: coral-credits
+coral_credits_chart_version: 0.1.0
+
+coral_credits_release_namespace: coral-credits
+coral_credits_release_name: coral-credits
+
+# The timeout to wait for coral credits API to become ready
+coral_credits_wait_timeout: 10m
+
+# The values for the release
+coral_credits_release_defaults: {}
+coral_credits_release_overrides: {}
```

Comment:
We need to make sure this requires the setting of the admin password, similar to what we do in other roles please.

Reply:
copy this style please:
https://github.com/stackhpc/ansible-collection-azimuth-ops/blob/500e98bc4ff8c93e8a75207f9908e483a737c866/roles/admin_dashboard_ingress/defaults/main.yml#L40

Reply:
We should do that in a style similar to Zenith, where we default to ingress being correctly configured, with TLS as appropriate:
https://github.com/stackhpc/ansible-collection-azimuth-ops/blob/500e98bc4ff8c93e8a75207f9908e483a737c866/roles/zenith/defaults/main.yml#L85-L143